site/src/AppRouter.tsx

```diff
@@ -123,16 +126,14 @@ export const AppRouter: FC = () => {
           }
         />
 
-        <Route path="workspaces">
-          <Route
-            index
-            element={
-              <AuthAndFrame>
-                <WorkspacesPage />
-              </AuthAndFrame>
-            }
-          />
-        </Route>
+        <Route
+          path="workspaces"
+          element={
+            <AuthAndFrame>
+              <WorkspacesPage />
+            </AuthAndFrame>
+          }
+        />
```

Comment:
What does this change do?

Reply:
Nothing, just simplify things. I was chasing a bug and I noticed this.